README.md

```diff
@@ -81,14 +92,23 @@ Set to 1 if you want the build script to always pull the base image when availab
 Use this variable in case you want to have a different context for your builds. By default
 the context of the build is the versioned directory the Dockerfiles are contained in.
 
+`SHELLCHECK_FILES`  
+One or more files or directories to be scanned by the shellcheck, default is `.`, which
+means a whole repository directory.
```

Comment:
Maybe explicitly mention that all shell files are scanned when a directory is provided. Not sure if we need it mentioned in the longer explanation above as well but it would not hurt.

```suggestion
means a whole repository directory. If a directory is provided then all of its content is scanned as well.
```